partiql-eval/src/eval.rs

```diff
@@ -627,39 +719,33 @@ impl Evaluator {
         // that all v ∈ V \{v0} are reachable from v0. Note that this is the definition of trees
         // without the condition |E| = |V | − 1. Hence, all trees are DAGs.
         // Reference: https://link.springer.com/article/10.1007/s00450-009-0061-0
-        match graph.externals(Incoming).exactly_one() {
-            Ok(_) => {
```

Comment:
This graph invariant no longer applies (e.g. could have two scan nodes without incoming nodes)